src/components/pages/address/address-events.tsx

```diff
@@ -1,13 +1,108 @@
-import { Address } from "viem";
-import { Card, CardHeader, CardContent } from "@/components/ui/card";
-
-const AddressEvents = ({ address }: { address: Address }) => {
-  return (
-    <Card>
-      <CardHeader>Latest 25 Contract Events</CardHeader>
-      <CardContent>CONTRACT EVENTS</CardContent>
```

Comment:
Move your table inside the card content and keep the header.
Also please move your table into its own component in a separate file.